.github/workflows/test.yml

```diff
@@ -9,20 +9,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        node-version: [20.x]
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
-      - run: npm ci # throws an error if the package-lock.json file is out of sync
-      - name: Check for linting errors (run `npm run lint:fix` to fix)
-        run: npm run lint
-      - name: Check for prettier errors (run `npm run format` to fix)
-        run: npx prettier --check .
```

Comment:
I'm guessing you've disabled this on purpose to make this PR easier to review, so thanks for that!